firmware/throttle/kia_soul_ps/throttle_control_module.ino

```diff
@@ -613,7 +617,7 @@ void loop()
         if ( current_ctrl_state.test_countdown >= 5 )
         {
             current_ctrl_state.test_countdown = 0;
-            check_spoof_voltages( &torque_spoof );
+            //check_spoof_voltages( &torque_spoof );
```

Comment:
Are we merging in commented code? I think removing it would be more in line with the style guide.